yt/yt/client/api/operation_client.h

```diff
@@ -361,6 +364,15 @@ struct TListJobsResult
     std::vector<TError> Errors;
 };
 
+struct TGetJobStderrResponse
+{
+    TSharedRef Data;
+    i64 TotalSize = 0;
+    i64 EndOffset = 0;
```

Comment:
Could please describe the exact semantic of `EndOffset`?

Reply:
Now ok ?